app/views/admin/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Welcome<%= ', ' + @current_user.name if @current_user %></h1>
```

Comment:
Can a non-admin reach this page? The conditional check might not be needed otherwise.

Reply:
I put this in before the auth check was implemented - solid catch 😄